core/column_descriptors.ts

```diff
@@ -84,6 +84,8 @@ export class ColumnDescriptors {
         return dataform.ColumnDescriptor.DimensionType.CATEGORY;
       case "timestamp":
         return dataform.ColumnDescriptor.DimensionType.TIMESTAMP;
+      case "number":
```

Comment:
Sorry - can we call this "numerical" ? Feels slightly more accurate, as can capture floats, doubles, ints, etc.

Reply:
I put `number` instead of `numerical` because we use `category` instead of `categorical`. I agree `numerical` sounds weird, but I can't figure out why